### PR TITLE
Add plugin info

### PR DESCRIPTION
**Pull request by iowillhoit**
### What does this PR do?
Adds the plugin-info command to display release notes on the command line

Do not merge this until https://github.com/salesforcecli/plugin-info/pull/33 and https://github.com/salesforcecli/plugin-info/pull/32 are merged.

### What issues does this PR fix or reference?
[@W-10260114@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-10260114)

### Testing Instructions
- Pull branch
- View help `./bin/run info releasenotes display --help`
- View notes `./bin/run info releasenotes display -v 1.3.0`
- Test alias `./bin/run whatsnew`

### CODE REVIEW

**Review by mdonnalley — CHANGES_REQUESTED** [review]
@iowillhoit I don't think we can add this as/is to `sf` since the help content won't match the sf style guide.

The easiest path forward is to create a `v2` branch of plugin-info with updated help.

Alternatively, we could write some code that will translate sfdx help into sf help

**Comment by iowillhoit** [comment]
@mdonnalley I know that `plugin-info` isn't using a markdown file, but the output is extremely close to a command that exists in `sf`. Top is `sf config list -help`, bottom is `sf whatsnew --help` What differences do you mean? 

Screen Shot 2021-12-01 at 2 09 35 PM

**Comment by mdonnalley** [comment]
@iowillhoit It's mostly going to be a matter of capitalization and punctuation: https://salesforce.quip.com/NJcjAyeE37gX#cSWACAsLARu

@jshackell-sfdc will be a better resource for that

**Comment by iowillhoit** [comment]
Gotcha, thanks @mdonnalley. I think rather than maintaining two versions of plugin-info it would be easiest to just use the `sf` help conventions in both `sf` and `sfdx`. I will sync with Juliet on this.

**Comment by iowillhoit** [comment]
Worked with @jshackell-sfdc to get the help docs in good shape
Screen Shot 2021-12-07 at 2 46 50 PM

File: package.json [hunk 1]
```diff
@@ -77,6 +85,7 @@
     "@oclif/plugin-help": "5.1.7",
     "@oclif/plugin-not-found": "2.2.0",
     "@oclif/plugin-plugins": "2.0.2",
+    "@salesforce/plugin-info": "^1.1.4",
```
Comment (mdonnalley) [inline package.json:88]: Can you alias this to match the other plugins? e.g. `@sf/info`

**Review by mdonnalley — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (iowillhoit) [reply package.json]: Updated 👍 Just curious, what benefits does this give aside from being less characters?

**Review by mdonnalley — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (peternhale) [reply package.json]: Allows one to use more than one version of the same package.

**Review by mdonnalley — APPROVED** [review]

**Review by RodEsp — APPROVED** [review]

**Pull request merged**